core/src/main/java/org/hibernate/ogm/datastore/impl/AvailableDatastoreProvider.java

```diff
@@ -20,7 +20,8 @@
 	MONGODB( "org.hibernate.ogm.datastore.mongodb.impl.MongoDBDatastoreProvider" ),
 	FONGO( "org.hibernate.ogm.datastore.mongodb.impl.FongoDBDatastoreProvider" ),
 	NEO4J_EMBEDDED( "org.hibernate.ogm.datastore.neo4j.impl.Neo4jDatastoreProvider" ),
-	COUCHDB_EXPERIMENTAL( "org.hibernate.ogm.datastore.couchdb.impl.CouchDBDatastoreProvider" );
+	COUCHDB_EXPERIMENTAL( "org.hibernate.ogm.datastore.couchdb.impl.CouchDBDatastoreProvider" ),
+	CASSANDRA( "org.hibernate.ogm.datastore.cassandra.impl.CassandraDatastoreProvider" );
```

Comment:
I will rename this into `CASSANDRA_EXPERIMENTAL` to reflect the current status (as we did for CouchDB).